.azure-pipelines.yml

```diff
@@ -43,6 +43,25 @@ variables:
     perf_tests: run
 
 jobs:
+  - job: configure
+    displayName: "Configure CI"
+    pool:
+      vmImage: "ubuntu-20.04"
+    steps:
+      - checkout: self
+        clean: true
+      - script: |
+          echo "Determine if any code has changed."
+          if git diff --ignore-submodules=dirty --quiet origin/main -- ':!docs' ':!*.md' ':!*.txt' ':!*.yml'; then
```

Comment:
This is hardcoding `origin/main`, but we want the target branch most of the time (which could be release/*). I'm looking for a way to do that.

Reply:
$SYSTEM_PULLREQUEST_TARGETBRANCH